_data/simple-icons.json

```
@@ -4560,6 +4560,11 @@
             "hex": "F86001",
             "source": "https://www.pjsip.org/favicon.ico"
         },
+        {
+            "title": "Planet",
+            "hex": "009DB1",
+            "source": "https://planet.com"
```

Let's change it and get this merged then! :tada: 

```suggestion
            "source": "https://www.planet.com/explorer/"
```

---

In the interests of getting this merged, I've committed this suggestion.